tests/components/profiler/test_init.py

```diff
@@ -211,9 +211,10 @@ def __repr__(self):
 
     assert hass.services.has_service(DOMAIN, SERVICE_DUMP_LOG_OBJECTS)
 
-    await hass.services.async_call(
-        DOMAIN, SERVICE_DUMP_LOG_OBJECTS, {CONF_TYPE: "DumpLogDummy"}, blocking=True
-    )
+    with patch("objgraph.by_type", return_value=[obj1, obj2]):
```

Comment:
Usually we would want to patch at the import point, but these get lazy loaded to reduce memory when the profiler is installed but not yet used